client/landing/gutenboarding/onboarding-block/design-selector/style.scss

```diff
@@ -1,268 +1,93 @@
-		position: absolute;
-		top: 0;
-		right: 0;
-		bottom: 0;
-		left: 0;
+	.design-selector__grid {
 		display: flex;
-		align-items: center;
-		justify-content: center;
-		opacity: 0;
-		transition: 400ms;
-
-		.design-selector__option-overlay-text {
-			font-size: 24px;
-			color: $white;
-			border-bottom: 2px solid $white;
-		}
+		flex-wrap: wrap;
+		justify-content: space-between;
 	}
 
-	&:hover {
-		.design-selector__option-overlay {
-			opacity: 1;
+	.design-selector__design-option {
+		width: 48%;
+		margin-bottom: 24px;
+
+		&.selected .design-selector__image-frame {
+			border: 2px solid var( --studio-green-20 );
 		}
 	}
-}
-
-// Adjust to display full with accounting for padding/borders
-.page-layout-selector__card-footer,
-.page-layout-selector__card-media {
-	display: block;
-	margin-left: -2px;
-	margin-right: -2px;
-}
 
-// Necessary for grid layout
-.page-layout-selector__card-footer {
-	background: $white;
-	bottom: 0;
-	position: absolute;
-	width: 100%;
-}
+	.design-selector__image-frame {
+		display: block;
+		height: 406px;
+		border: 1px solid var( --studio-gray-5 );
+		border-radius: 4px;
+		overflow: hidden;
+		margin: 8px;
+		transition: transform 0.2s;
+		box-shadow: rgba( 0, 0, 0, 0.2 ) 0 0 0;
 
-// Necessary for grid layout
-.page-layout-selector__card-media {
-	position: absolute;
-	top: 0;
-	width: 100%;
-}
+		&:hover {
+			transform: scale( 1.03 );
+			box-shadow: rgba( 0, 0, 0, 0.2 ) 0 0 15px;
+		}
 
-.design-selector__description-container {
-	position: fixed;
-	display: flex;
-	flex-direction: column;
-	width: calc( 50% - #{$design-selector-grid-gap} );
-	padding-top: 23px;
 
-	@include breakpoint( '>660px' ) {
-		&.on-right-side {
-			right: 0;
+		@include breakpoint( '<1400px' ) {
+			height: 326px;
 		}
-	}
-}
 
-.design-selector__description-title {
-	color: $dark-gray-800;
-	font-size: 2em;
-	font-weight: bold;
-	margin-bottom: 0.8em;
-}
-.design-selector__description-description {
-	color: $dark-gray-800;
-	font-size: 1.3em;
-	line-height: 1.3;
-}
-
-.accessible-focus {
-	.page-layout-selector__item {
-		&:focus {
-			box-shadow: 0 0 0 1px $white, 0 0 0 3px $blue-medium-500;
-			// Windows High Contrast mode will show this outline, but not the box-shadow.
-			outline: 2px solid transparent;
+		@include breakpoint( '<1280px' ) {
+			height: 243px;
 		}
-	}
 
-	.design-selector__design-option {
-		&:focus {
-			.design-selector__option-overlay {
-				opacity: 1;
-			}
+		@include breakpoint( '<660px' ) {
+			height: 163px;
 		}
 	}
-}
 
-// When page selector is open, design card should behave like a static image.
-.design-selector__grid-container {
-	&.is-page-selector-open {
-		.design-selector__design-option {
-			cursor: default;
-		}
-		.design-selector__option-overlay {
-			display: none;
-		}
+	.design-selector__option-name {
+		margin-top: 16px;
+		display: block;
+		text-align: center;
+		width: 100%;
+		font-size: 24px;
+		font-family: Recoleta, Georgia, 'Times New Roman', Times, serif;
```

Comment:
After rebase, we can now pull this from a mixin in Gutenboarding's root folder.

Reply:
Done.